frontend/pubspec.yaml

```diff
@@ -26,6 +26,7 @@ dependencies:
   intl: ^0.18.1
   webview_flutter: ^4.4.2
   shared_preferences: ^2.2.2
+  flutter_form_builder: ^9.2.0
```

Comment:
This dependency is optional for this. We can build this form without it with likely only a bit more code. It's good for this state, but likely can be removed in the future if desired.

Reply:
yeah I was wondering if it was overkill - might be easier to do without.

Reply:
removed